Ble adv device name (#684)
nrfconnect/sdk-nrf-bm: samples/bluetooth/ble_bms/README.rst

#. Reset the kit.
#. In the Serial Terminal, observe that the ``BLE BMS sample initialized`` message is printed.
#. Observe that the ``Advertising as nRF_BM_BMS`` message is printed.
   You can configure the advertising name using the :kconfig:option:`CONFIG_SAMPLE_BLE_DEVICE_NAME` Kconfig option.
   For information on how to do this, see `Configuring Kconfig`_.
#. Open `nRF Connect for Desktop`_.
#. Open the `Bluetooth Low Energy app`_ and select the connected device that is used for communication.
#. Connect to the device from the app.

>    If the device is not advertising, you might need to use the :guilabel:`Reset Board` option in |VSC| or press the reset button on the board.

Reviewer: ```suggestion
   If the device is not advertising, you can use the :guilabel:`Reset Board` option in |VSC| or press the reset button on the board.
```
Or "you may need to"

Or simply:
```suggestion
   If the device is not advertising, use the :guilabel:`Reset Board` option in |VSC| or press the reset button on the board.
```
